Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -123,6 +123,14 @@
             "required": true,
             "visibleRule": "updateType = Docker image settings",
             "helpMarkDown": "Path to a text file that contains the digest values of the Docker images associated with the Service Fabric application. This file can be output by the [Docker task](https://go.microsoft.com/fwlink/?linkid=848006) when using the push action. The file should contain lines of text in the format of 'registry/image_name@digest_value'."
+        },
+        {
+            "name": "overwriteExistingPkgArtifact",
+            "type": "boolean",
+            "label": "Overwrite Existing Package Artifact",
+            "defaultValue": "true",
+            "required": false,
+            "helpMarkDown": "Always download a new copy of the artifact. Otherwise use an existing copy, if present."
```

Comment:
Meant to add this comment earlier. You need to add the property:
            "visibleRule": "updateType = Manifest versions && updateOnlyChanged = true" Please place this input with the As this should only be used when updating versions based on changes. Also move this input above imageDigestsPath so it is grouped with the others about updating versions.